appveyor.yml

```diff
@@ -0,0 +1,115 @@
+os: Visual Studio 2013
+
+shallow_clone: true
+
+environment:
+  CONFIG: Release
+  matrix:
+    - BUILD: Kodi
+    - ADDONS: adsp
+    - ADDONS: audiodecoder
+    - ADDONS: audioencoder
+    - ADDONS: pvr
+    - ADDONS: screensaver
+    - ADDONS: visualization
+
+install:
+  - SET PATH=C:\Program Files (x86)\CMake\bin;C:\msys64\bin;C:\msys64\usr\bin;%PATH%
+  - bash -lc "pacman --needed --noconfirm -Sy"
+  - bash -lc "pacman --needed --noconfirm -S curl"
+  - mklink /j %APPVEYOR_BUILD_FOLDER%\project\BuildDependencies\msys64 C:\msys64
+
+build_script:
+  # Setup Visual Studio compiler environment 
+  - ps: $ErrorActionPreference= 'silentlycontinue'
+  
+  # Setup Visual Studio compiler environment 
+  - call "%VS120COMNTOOLS%..\..\VC\bin\vcvars32.bat"
+  
+  #
+  # Build Kodi
+  #
+  - ps: |
+      if ($env:BUILD -eq "Kodi") {
+        # Download dependencies
+        cd $env:APPVEYOR_BUILD_FOLDER\project\BuildDependencies
+        & .\DownloadBuildDeps.bat
+        
+        # We are using AppVeyor's already installed msys64 environment,
+        # no need to install msys/mingw environment.
+        
+        # Download precompiled mingw32 libraries
+        # To run make-mingwlibs.bat on AppVeyor takes too long, 
+        # we must use precompiled libs to speed up build
+        bash -c "curl http://repo.msys2.org/mingw/i686/mingw-w64-i686-ffmpeg-2.8.5-2-any.pkg.tar.xz | tar xJv"
+        bash -c "curl http://repo.msys2.org/mingw/i686/mingw-w64-i686-libdvdcss-1.4.0-1-any.pkg.tar.xz | tar xJv"
+        bash -c "curl http://repo.msys2.org/mingw/i686/mingw-w64-i686-libdvdnav-5.0.3-1-any.pkg.tar.xz | tar xJv"
+        bash -c "curl http://repo.msys2.org/mingw/i686/mingw-w64-i686-libdvdread-5.0.3-1-any.pkg.tar.xz | tar xJv"
```

Comment:
It's strange what build was successuf, because we export a few methods which are private in original.